applications/nrf5340_audio/src/bluetooth/le_audio.h

```diff
@@ -53,32 +64,26 @@
 #else
 #error Unsupported LC3 codec preset for unicast
 #endif /* CONFIG_BT_AUDIO_UNICAST_RECOMMENDED */
-#endif /* CONFIG_AUDIO_SOURCE_USB */
 #endif /* CONFIG_TRANSPORT_CIS */
```

Comment:
Seems that there is some mix between UNICAST and CIS. We should to a larger extent stick with one of them.

Reply:
Agree, but maybe not in this PR